mappings/net/minecraft/world/biome/source/util/OverworldBiomeParameters.mapping

```diff
@@ -0,0 +1,122 @@
+	FIELD f_ksqbybun erosions [Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_nipjbvyl mushroomFieldsContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_pwzmxluc regularBiomes [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_qfzszkny deepOceanContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_rhpdqupa frozenTemperature Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_rvcyduuk plateauBiomes [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_snbdsnqj oceanContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_ugfrbvdx oceanBiomes [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_vbpnuepb farInlandContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_vurbgrlc PEAK_END F
+	FIELD f_wrwcwcob inlandContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_wzzxoteg midInlandContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_xpcohlkz regularBiomesVariant [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_ywphpjca humidities [Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_zddmujpv PEAK_START F
+	METHOD m_agfvgbns pickBeachBiome (II)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+	METHOD m_aluorree addBiomesTo (Ljava/util/function/Consumer;)V
+		ARG 1 biomeEntryConsumer
+	METHOD m_asupgdcz addUndergroundBiomesTo (Ljava/util/function/Consumer;)V
+		ARG 1 parameters
+	METHOD m_awfztaxz addMidSliceTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)V
+		ARG 1 parameters
+		ARG 2 weirdness
+	METHOD m_byldhmuc pickPeakBiome (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_cbutysev pickRegularBiomeOrBadlandsIfHotOrSlopeIfCold (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_cdojkptw addValleysTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)V
+		ARG 1 parameters
+		ARG 2 weirdness
+	METHOD m_ejtjjzrz getTemperatureDescription (D)Ljava/lang/String;
+		ARG 1 temperature
+	METHOD m_ekzzkomy addOffCoastBiomesTo (Ljava/util/function/Consumer;)V
+		ARG 1 parameters
+	METHOD m_ezyzmjyg pickHillBiome (IILnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+		ARG 3 weirdness
+	METHOD m_fgargpsb getPeaksAndValleysDescription (D)Ljava/lang/String;
+		ARG 0 erosion
+	METHOD m_gaoknfao addUndergroundBiomeTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;FLnet/minecraft/unmapped/C_hcaelhhx;)V
```

Comment:
`addCaveBiomeTo`? That would be a simpler name.

Reply:
I don't know how much accurate either one is.

Reply:
Both are definitely correct, it's not like one is wrong. "Cave" is just simpler than "underground", and it's only used for cave biomes at the moment.
But this doesn't really matter.

Reply:
underground is more about intent than cave. this is similar to the mojmap name, which i believe shows that the intent is correct and is clearer to modders what it does.